frontend/src/views/JobTrackingView.vue

```diff
@@ -158,13 +158,19 @@ export default {
     async showDetailModal(job) {
       if (job) {
         // editing job
-        await postRequest('job/api/get_job_by_id', {
-          user_id: this.activeUser.id,
-          job_id: job.id,
-        }).then((completeJob) => {
-          this.selectedJob = completeJob.job_data
+        if (!job.deadlines) {
```

Comment:
this could be any of the non-minimal fields e.g. `cover_letter`, `description`, etc.